templates/tools/dockerfile/python_pip_builds.include

```diff
@@ -0,0 +1,10 @@
+#===================================
+# Install Python build requirements
+RUN /opt/python/cp36-cp36m/bin/pip install --upgrade 'cython<3.0.0rc1'
```

Comment:
We're using cython 3.0 now, this doesn't have to be included in this PR, can you add a TODO here to update this?

Reply:
had to update it in this PR itself as builds were failing with the old version